pycbc/vetoes/chisq.py

```diff
@@ -335,11 +335,16 @@ def cached_chisq_bins(self, template, psd):
             psd._chisq_cached_key[id(template.params)] = True
             num_bins = int(self.parse_option(template, self.num_bins))
```

Comment:
I won't hold this up if you want to merge it urgently, but I don't believe this is the best solution. 

The issue would seem to be thta the sigmas_vec cache itself was not generated with a low enough flow to cover the range of flows that might be encountered. The solution would seem to be to generate the cache from the lowest flow in the bank (it can know this). 

The solution here adds some extra complexity but also causes the cache to be regenerated for every template + PSD combo (unless some templates have flow the same). The only other saving would be if some data segments use the same PSD, but that also is not used to a high degree.

Reply:
I think the better fix would be to use the bank flow here (self.min_f_lower)
https://github.com/gwastro/pycbc/blob/master/pycbc/waveform/bank.py#L63

and then take a difference of kmin / kmax here instead of just kmax
https://github.com/gwastro/pycbc/blob/master/pycbc/waveform/bank.py#L74

Reply:
Agreed. I've changed the patch to do this now.

(As the templates have lower frequencies rounded to the nearest 0.1Hz, there's not a large number (O(10)) of different low-frequency cutoffs used for the SPATmplt templates, but I agree it's better not to compute the same thing multiple times.)